app/controllers/evaluations_controller.rb

```diff
@@ -85,6 +85,23 @@ def update
     end
   end
 
+  def recuse
+    fetch_evaluator_submission_assignment
+    return unauthorized_redirect unless can_access_evaluation?
+
+    begin
+      if recuse_evaluator
+        destroy_recused_evaluation
+        flash[:notice] = I18n.t("evaluations.recusal.success")
+        redirect_to submissions_evaluation_path(@evaluator_submission_assignment.phase)
```

Comment:
my browser is giving a 404 not found and redirecting. I found the documentation actually [mentions this issue](https://edgeapi.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to) with redirecting from an XHR PATCH and the solution is to return a 303 status instead of 302.

> If you are using XHR requests other than GET or POST and redirecting after the request then some browsers will follow the redirect using the original request method. This may lead to undesirable behavior such as a double DELETE. To work around this you can return a 303 See Other status code which will be followed using a GET request.

```suggestion
        redirect_to submissions_evaluation_path(@evaluator_submission_assignment.phase), status: 303
```